packages/language/src/workspace/file-store.ts

```diff
@@ -31,6 +31,14 @@ export class FileStore {
     return this.get(uri)?.tokens;
   }
 
+  getAllTokens(): Token[] {
+    const allTokens: Token[] = [];
+    for (const file of this.map.values()) {
+      allTokens.push(...file.tokens);
+    }
+    return allTokens;
+  }
```

Comment:
Suggestion: Since we iterate over this anyway, can we use a generator function? Otherwise we probably generate a lot of unnecessary arrays.
```suggestion
  *getAllTokens(): IterableIterator<Token> {
    for (const file of this.map.values()) {
      yield* file.tokens;
    }
  }
```